src/CoreWCF.Http/tests/CoreWCF.Http.Tests.csproj

```diff
@@ -6,18 +6,24 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+
+    <!-- Required for multi-framework support for in-memory integration testing-->
+    <OutputType>Exe</OutputType>
+    <GenerateProgramFile>false</GenerateProgramFile>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Is PreserveCompilationContext needed? Reading docs suggests this is only needed if you are needing to compile Razor views and there aren't any in this project.

Reply:
It's required by the `TestServer`, which makes use of the context in the bowels somewhere, to wire up the in-memory version of the HTTP stack.